### PR TITLE
Repo cleanup part 1 & 2

### PR DESCRIPTION
**Pull request by eleanor-byhook**
Remove unused files (utils, .swp file) and references.
Refactor Control-panel to be more independent of the tweening element and self contained.
Update readme to reflect refactor

### CODE REVIEW

**Comment by JeremiahReichardt** [comment]
I would remove the wording about python's simple server as Windows doesn't ship with Python installed ( that I know of anyways )

**Comment by eleanor-byhook** [comment]
How would you run it on Windows?  Maybe I could add instructions for that and add a "On Mac:" for this instruction

**Comment by JeremiahReichardt** [comment]
There are a bunch of ways, it just needs to run via http:// and not file://

https://github.com/PolymerLabs/polyserve is a good option

File: README.md [hunk 1]
```diff
@@ -1,43 +1,39 @@
...
+  ```
+  ready: function() {
+    this.timeline = new TimelineMax({ repeat: -1});
+    this.timeline.pause();
+    this.timeline.add( new TweenLite.to( this.$$('#whale'), 1, {x: '701px'}), 0);
+  },
+  ```
+
+
+3b. Optionally if you wish to use the repeat delay funtionality, make sure the
```
Comment (JeremiahReichardt) [inline README.md:35]: 3b is weird, just use 4

**Pull request merged**